src/NLog.Extensions.Configuration/NLog.Extensions.Configuration.csproj

```diff
@@ -12,7 +12,10 @@
 
     <Company>NLog</Company>
     <Authors>Julian Verdurmen;CoCo Lin</Authors>
-    <Description>NLog extensions for Microsoft.Extensions.Configuration</Description>
+    <Description>NLog extensions for Microsoft.Extensions.Configuration
+
+Read appsettings.json on Net Core using ${configsetting:name=MySetting}
```

Comment:
I prefer writing .NET instead of Net and keep it consistent everywhere. 

I think it's everywhere the platforms are written as:

- .NET
- .NET Standard
- .NET Core
- ASP.NET
- ASP.NET Core

Reply:
Done